### DISCUSSION TITLE
Great Vault reminder

### DISCUSSION THREAD

**Discussion by TheSoulRender** [SavedInstances/SavedInstances#536] [Ideas]
Hi, I don't know if it's already doable in the add-on, I've looked regularly over the last few months and never found the option, would there be a way to add a great vault reminder to the add on? To show you if a character has an item to take from the great vault & if they have taken it already since reset.

I often do dungeon or raid runs on my alts but never remember on which, and every week on reset I have to check the vault on all my characters despite most having no loot to obtain from it, which is quite annoying... Any way to do that ? <3

**Comment by LiangYuxuan** [comment]
It is currently supported by Mythic Plus Best
